lib/safe-pg-migrations/plugins/statement_insurer.rb

```diff
@@ -46,8 +46,25 @@ def add_foreign_key(from_table, to_table, **options)
       without_statement_timeout { validate_foreign_key from_table, options_or_to_table } unless validate_present
     end
 
+    def create_table(*)
+      with_setting(:statement_timeout, SafePgMigrations.config.pg_safe_timeout) do
+        super do |td|
+          yield td if block_given?
+          td.indexes.map! do |key, index_options|
+            index_options[:algorithm] ||= nil
```

Comment:
I'd name it `:default` instead of `nil`, to make the check in `add_index` more straightforward

Reply:
Good point, applied